ludwig/utils/trainer_utils.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+
+def get_terminal_steps_per_checkpoint(
```

Comment:
not sure i like the word terminal here. At the beginning i thought it meant they were used only in the terminal, so maybe it's a bit confusing.
Maybe final_steps_per_checkpoint could be better, or maybe computed_steps_per_checkpoint, with the function called compute_steps_per_checkpoint '